.github/workflows/pac.yml

```diff
@@ -0,0 +1,41 @@
+name: Policy as Code
+
+on:
+  pull_request:
+    paths:
+      - ci/repo-policy/**.tf
+
+env:
+  TERRAFORM_DIR: "./ci/repo-policy"
+  GITHUB_TOKEN: ${{ secrets.ITS_GH_TOKEN }}
+
+jobs:
+  terraform:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::754489498669:role/gromit-ci
+          role-session-name: gromitci
+          aws-region: eu-central-1
+
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.3.0
+
+      - name: Terraform Init
+        working-directory: ${{ env.TERRAFORM_DIR }}
+        id: init
+        run: terraform init -input=false
+
+      - name: Terraform Plan
+        working-directory: ${{ env.TERRAFORM_DIR }}
+        id: plan
+        run: |
+          terraform validate && terraform plan
```

Comment:
Since TFC is not working as hoped, the DX here needs to be reasonable. It would be good to use https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines to group the plan output so that it can be easily located.
Plan output is not easy to read. There are some tools, googling turned up https://hieven.github.io/terraform-visual/ which could be added to the pipeline so that it is easy to see what the plan is going to do.